source/administration/upgrade.rst

```diff
@@ -158,7 +158,7 @@ After the server is upgraded, users might need to refresh their browsers to expe
 
       cd {install-path}/mattermost
       sudo mv plugins~/ plugins
-      sudo mv client/plugins~/ mattermost/client/plugins
+      sudo mv client/plugins~/ client/plugins

```

Comment:
Should we instead make this step consistent with step 8, i.e.

```
cd {install-path}/mattermost
sudo mv plugins~/ plugins
sudo mv client/plugins~/ client/plugins
```

becomes

```
sudo mv mattermost/plugins~/ mattermost/plugins
sudo mv mattermost/client/plugins~/ mattermost/client/plugins
```

OR alternatively, if this is cleaner, should we make similar changes for step 8 by cd:ing into the mattermost directory first?

Reply:
Sure. Up to you :)

Reply:
I was looking for a consistent pattern in the upgrade guide, but each step does something different..some cd to `{install-path}`, some cd to `{install-path}/mattermost` and some don't cd at all..need to fix this..

So let's go with your proposal here, which is most consistent for steps 11 onward.